src/backend/bottles/utils.ts

```diff
@@ -0,0 +1,114 @@
+import { spawn } from 'child_process'
+import { logDebug, LogPrefix } from '../logger/logger'
+import { isFlatpak } from '../constants'
+import shlex from 'shlex'
+
+function prepareBottlesCommand(
+  args: string[],
+  bottlesBin: string,
+  noCli?: boolean,
+  join?: boolean
```

Comment:
```suggestion
  noCli = false,
  join = false
```
This is mostly preference, but it also makes it clear to whoever is calling this function what the default values are (yes, "missing -> false/disabled" might be obvious, but it's one more thing to think about)